S26-documentation/07c-tables.t

```diff
@@ -0,0 +1,68 @@
+use v6;
+
+use MONKEY-SEE-NO-EVAL;
+use Test;
+
+my $f = './.tmp-test-file';
+
+my $code = gen-test($f);
+
+EVAL $code;
+
+##### subroutines #####
+sub gen-test($f) {
+    my $fh = open $f, :w;
+
+    $fh.print: q:to/HERE/;
+    use v6;
+
+    use Test;
+
+    plan 8;
+    my $r;
+
+    # table with no ending ws
+    =table
+    HERE
+
+    $fh.say: "    X | O |";
+
+    $fh.print: q:to/HERE/;
+       ---+---+---
+          | X | O
+       ---+---+---
+          |   | X
+
+
+    # table with ending ws
+    =table
+    HERE
+
+    $fh.say: "    X | O | ";
+
+    $fh.print: q:to/HERE/;
+       ---+---+---
+          | X | O
+       ---+---+---
+          |   | X
+
+
+    $r = $=pod[0];
+    is $r.contents.elems, 3;
+    is $r.contents[0].join(','), 'X,O,';
```

Comment:
I'm not sure why you use `.join(',')` here instead of using is-deeply and testing on an array?

Reply:
Well, I've been using that construct for tables since I started because existing tests used it.  It seems natural because one can see easily what the table row results should be.